components/edit-collective/sections/Tiers.js

```diff
@@ -76,9 +72,7 @@ const CardsContainer = styled(Grid).attrs({
  */
 const Tiers = ({ collective }) => {
   const variables = { accountSlug: collective.slug };
-  const { data, loading, error, refetch } = useQuery(listTierQuery, { variables, context: API_V2_CONTEXT });
-  const tiers = sortBy(get(data, 'account.tiers.nodes', []), 'legacyId');
-  const filteredTiers = collective.type === 'EVENT' ? tiers.filter(tier => tier.type !== 'TICKET') : tiers; // Events have their tickets displayed in the "Tickets" section
```

Comment:
This is not right; we should be using data from this query, not `collective.tiers` which may not exist in the future, or be outdated

Reply:
Okay, makes sense. I can re-establish it, ok?

Reply:
Fetching of tiers was reestablished. So a small challenge, the tier model between `collective.tier` and `account.tiers.nodes` is not the same. The tier-sorting in collective.settings uses the integer id of `collective.tier`  to manage order, while in `account.tiers.nodes`, id is an uuid, and the old integer id is in `tier.legacyId`. I've fixed this by duck-typing in `tier-utils.js`. But an alternative would be to refactor `pages/contribute.js` to fetch the account.tiers.nodes instead of using the collective.tiers .... I'm not sure what is best, though.

Reply:
Yes, this is because one comes from our legacy GraphQL V1 API, the other [from GraphQL V2](https://graphql-docs-v2.opencollective.com/welcome)